test/Clownface/constructor.test.js

```diff
@@ -7,14 +7,14 @@ import Clownface from '../../lib/Clownface.js'
 describe('constructor', () => {
   it('should create a Clownface object', () => {
     const dataset = rdf.dataset()
-    const cf = new Clownface({ dataset })
+    const cf = new Clownface({ dataset, factory: rdf })
```

Comment:
This seems like a breaking change but I find the Class internal, thus the change should not affect anyone